src/editor-sidebar/metadata-editor-modal.js

```diff
@@ -177,16 +182,12 @@ export const ThemeMetadataEditorModal = ( { onRequestClose } ) => {
 						'create-block-theme'
 					) }
 				/>
-				<TextareaControl
+				<FormTokenField
 					label={ __( 'Theme tags', 'create-block-theme' ) }
-					value={ theme.tags_custom }
-					onChange={ ( value ) =>
-						setTheme( { ...theme, tags_custom: value } )
+					value={
+						theme.tags_custom ? theme.tags_custom.split( ', ' ) : []
 					}
-					placeholder={ __(
-						'A comma-separated collection of tags',
-						'create-block-theme'
-					) }
+					onChange={ onChangeTags }
 				/>
```

Comment:
I agree with @pbking in that we should include a link to the available tags documentation. The `FormTokenField` component unfortunately doesn't have a `help` prop like the other components used here, but we can add the link regardless:


```js
<>
	<FormTokenField
		label={ __( 'Theme tags', 'create-block-theme' ) }
		value={
			theme.tags_custom
				? theme.tags_custom.split( ', ' )
				: []
		}
		onChange={ onChangeTags }
	/>
	<ExternalLink
		href="https://make.wordpress.org/themes/handbook/review/required/theme-tags/"
		style={ {
			fontSize: '12px',
			marginTop: '-20px',
			marginBottom: '1rem',
		} }
	>
		{ __( 'Read more.', 'create-block-theme' ) }
	</ExternalLink>
</>
```

I've tried a bottom margin the re as well, in order to more or less match the spacing between the link and the next header in the other sections.

<img width="612" alt="image" src="https://github.com/WordPress/create-block-theme/assets/1157901/f57f9d10-4c4c-4a12-8628-085c96eff5da">

Reply:
Thanks @vcanales.

I pulled in the changes you suggested.